tools/export_slow_tests.py

```diff
@@ -37,11 +38,38 @@ def filter_slow_tests(test_cases_dict: Dict[str, float]) -> Dict[str, float]:
     return {test_case: time for test_case, time in test_cases_dict.items() if time >= SLOW_TEST_CASE_THRESHOLD_SEC}
 
 
-def export_slow_tests(filename: str) -> None:
+def get_test_infra_slow_tests() -> Dict[str, float]:
+    url = "https://raw.githubusercontent.com/pytorch/test-infra/master/stats/slow-tests.json"
+    contents = urlopen(url, timeout=1).read().decode('utf-8')
+    return cast(Dict[str, float], json.loads(contents))
+
+
+def too_similar(calculated_times: Dict[str, float], other_times: Dict[str, float], threshold: float) -> bool:
+    # check that their keys are the same
+    if calculated_times.keys() != other_times.keys():
+        return False
+
+    for test_case, test_time in calculated_times.items():
+        if test_case not in other_times:
```

Comment:
isn't this not possible given the `calculated_times.keys() != other_times.keys():` check above?

Reply:
Ah yes, I forgot to remove debugging code oops!